lib/NetPacks.h

```diff
@@ -1545,6 +1545,7 @@ struct BattleStackMoved : public CPackForClient
 		h & stack;
 		h & tilesToMove;
 		h & distance;
+		h & teleporting;

```

Comment:
Please, consider making backward compatibility

Reply:
Marking as resolved.
Unfortunately don't see a way to add compatibility for all *netpacks*, so multiplayer between 1.1 & 1.2 will be broken.
Save compatibility with 1.1 should be fixed.

Reply:
as offtop, we should introduce separate version for net packs